common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -252,6 +252,11 @@ typedef struct {
 } sHOSTAP_ENABLED_NOTIFICATION;
 
 #define SSID_MAX_SIZE beerocks::message::WIFI_SSID_MAX_LENGTH
+// The absolute maximum size of any frame according to the 802.11 specification
+// is MMPDU size of 2304 bytes. The actual size of an (re)association frame
+// should be in the range of a couple of hundreds of bytes, so be on the safe side
+// and set the maximum size to 2KB
```

Comment:
Move this explanation to the place where the number 2048 is used.

Reply:
Oh here it is!

Reply:
it used here - ASSOCIATION_MAX_LENGTH is defined to 2048 in beerocks_defines.h, do you want me to move this comment there?

Reply:
Yes, that's what we mean.

The comment explains why the value 2048 is chosen, but that value is not defined here, it's defined in beerocks_defines.h.

Reply:
done